src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -199,6 +204,20 @@ public virtual Publisher.IPublisherClient GrpcClient
             get { throw new NotImplementedException(); }
         }
 
+        /// <summary>
+        /// Creates the given topic with the given name.
+        /// </summary>
+        /// <param name="name">
+        /// The name of the topic. It must have the format
+        /// `"projects/{project}/topics/{topic}"`. `{topic}` must start with a letter,
+        /// and contain only letters (`[A-Za-z]`), numbers (`[0-9]`), dashes (`-`),
+        /// underscores (`_`), periods (`.`), tildes (`~`), plus (`+`) or percent
+        /// signs (`%`). It must be between 3 and 255 characters in length, and it
+        /// must not start with `"goog"`.
```

Comment:
This will be validated at some point no? Seems oddly specific to have this regexp in the comments, just wondering if it is not better to point users to the documentation. As in `The name must be valid according to the specification at...`.